src/main/java/inf112/skeleton/app/interfaces/IRobot.java

```diff
@@ -1,19 +1,19 @@
 package inf112.skeleton.app.interfaces;
-// RT initial setup jan 29/30 - 2019
+
+import inf112.skeleton.app.enums.Direction;
+
+// RT initial setup jan 29/30 - 2019 - cont feb 12
 public interface IRobot {
 
-    public void move();         // int = -1,1,2,3 (steps)
-    public void turn();         // int = 90, 270, 180 degrees
-    public void getDirection(); // get current direction of robot as int (0, 90, 280, 270)
-    public void giveCards();    // give chosen object(with card ID) of 5 cards to robot, to put in RobotMemory
```

Comment:
Should this method have a parameter. For example giveCards(List<IProgramCard> cards); or something like that?

Reply:
No should not be abstract.
All the above from IRobot is removed, as functionallity of moving robot is in board, also giveCard, as we discussed that maybe cards to program robot is in deck-object in user.
Above routines should be changed in branch, are they not?

Reply:
Oh, lol, did'nt realize that :laughing:

Reply:
I approve if you remove abstract ;)